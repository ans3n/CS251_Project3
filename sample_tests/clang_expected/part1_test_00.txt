bucket_count
9
size
0
empty
true
insert V6woTC -520.19
peek V6woTC
-520.19
size
1
hash_code V6woTC
8
insert ZhpLUu -184.155
insert NE3X3fb -47.0941
bucket_count
9
print
  0: (0) NE3X3fb -> -47.0941
  1: [empty]
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: (7) ZhpLUu -> -184.155
  8: (8) V6woTC -> -520.19
peek V6woTC
-520.19
peek NE3X3fb
-47.0941
peek ZhpLUu
-184.155
size
3
print
  0: (0) NE3X3fb -> -47.0941
  1: [empty]
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: (7) ZhpLUu -> -184.155
  8: (8) V6woTC -> -520.19
insert SpMNL9w 357.968
insert nQpUXt 192.883
print
  0: (0) NE3X3fb -> -47.0941
  1: (6) nQpUXt -> 192.883
  2: [empty]
  3: [empty]
  4: [empty]
  5: [empty]
  6: (6) SpMNL9w -> 357.968
  7: (7) ZhpLUu -> -184.155
  8: (8) V6woTC -> -520.19
insert VD0xu1a 628.569
insert iw5Ad -503.044
quit
