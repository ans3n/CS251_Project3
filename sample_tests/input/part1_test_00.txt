string float
9
bucket_count
size
empty
insert V6woTC -520.19
peek V6woTC
size
hash_code V6woTC
insert ZhpLUu -184.155
insert NE3X3fb -47.0941
bucket_count
print
peek V6woTC
peek NE3X3fb
peek ZhpLUu
size
print
insert SpMNL9w 357.968
insert nQpUXt 192.883
print
insert VD0xu1a 628.569
insert iw5Ad -503.044
quit
