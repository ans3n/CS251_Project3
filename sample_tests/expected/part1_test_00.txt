bucket_count
9
size
0
empty
true
insert V6woTC -520.19
peek V6woTC
-520.19
size
1
hash_code V6woTC
5
insert ZhpLUu -184.155
insert NE3X3fb -47.0941
bucket_count
9
print
  0: [empty]
  1: [empty]
  2: (2) ZhpLUu -> -184.155
  3: [empty]
  4: (4) NE3X3fb -> -47.0941
  5: (5) V6woTC -> -520.19
  6: [empty]
  7: [empty]
  8: [empty]
peek V6woTC
-520.19
peek NE3X3fb
-47.0941
peek ZhpLUu
-184.155
size
3
print
  0: [empty]
  1: [empty]
  2: (2) ZhpLUu -> -184.155
  3: [empty]
  4: (4) NE3X3fb -> -47.0941
  5: (5) V6woTC -> -520.19
  6: [empty]
  7: [empty]
  8: [empty]
insert SpMNL9w 357.968
insert nQpUXt 192.883
print
  0: [empty]
  1: [empty]
  2: (2) ZhpLUu -> -184.155
  3: [empty]
  4: (4) NE3X3fb -> -47.0941
  5: (5) V6woTC -> -520.19
  6: (6) nQpUXt -> 192.883
  7: (7) SpMNL9w -> 357.968
  8: [empty]
insert VD0xu1a 628.569
insert iw5Ad -503.044
quit
